bin/completed_segments.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+""" Find (and delete) fully completed kafka-backup segments """
+
+import os
+import re
+import sys
+import argparse
+
+
+def filename_pattern():
+    """ Kafka-backup segment/index filename pattern """
+    return 'segment_partition_%s_from_offset_%s_%s'
+
+
+def parse_args():
+    """ Parse cmdline args """
+    parser = argparse.ArgumentParser(
+        description='Find (and delete) fully completed kafka-backup segments'
+    )
+    parser.add_argument(
+        '-d', '--delete',
+        help='Delete fully completed segment files',
+        action='store_true',
+        default=False,
+    )
+    parser.add_argument(
+        'target_dir',
+        help='Backup directory (target.dir)',
+        default=os.getcwd(),
+    )
+    return parser.parse_args()
+
+
+def collect_segments(target_dir):
+    """ Collect segment partitions & offsets per directory (topic) """
```

Comment:
could you please document how the result looks like?

Reply:
ok, will add data structure to comment here